CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 Current Version
-  - Added CODEOWNERS
+  - Add `Offers` resource
 
 2.4.0 December 6, 2017
   - Add `Links` resource
+  - Added CODEOWNERS
```

Comment:
It looks like this wasn't a part of 2.4.0, it was after.

Reply:
I think there was an error in the CHANGELOG.

It was in `Current Version` which would in the current state is 2.5.0, even though this release would be be 2.5.0.

Happy to arrange this as you see fit